server/middlewares/verifications/tripVerification.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable consistent-return */
+import tripService from '../../services/tripServices';
+import reqService from '../../services/tripReqService';
+import Util from '../../helpers/util';
+import { debugLogger }from '../../helpers/loggerHandle';
+
+const util = new Util();
+
+export const tripVerification = async(req, res, next) =>{
+    const oldTripDetails = await tripService.findByProp({id: req.params.trip_id});
+
+    if(!oldTripDetails[0]){
+      const Error = 'This trip is not found in the database.';
+        debugLogger(req, 404, Error);
+        util.setError(404, Error);
+        return util.send(res);
+    }
+  
+    req.oldTripDetails = oldTripDetails[0].dataValues;
```

Comment:
@irfiacre you may refer to the comment given above regarding the use of the old trip details.

Reply:
Here also it's the same.